packages/dds/tree/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@fluidframework/tree",
```

Comment:
Since this package is not ready for consumption, we should include a `"private": true` to make it excluded from publishing by lerna.

Reply:
Done.

Reply:
Turns out marking the package as private causes [Policy-Check to fail](https://dev.azure.com/fluidframework/public/_build/results?buildId=60066&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=baa7d181-d9ee-513a-8016-f44a98fd040e&l=38).

@tylerbutler - Do you have suggestions on what to do here? Just not make the package private?

Reply:
CC @DLehenbauer for visibility since you liked the suggestion to have it be private.

Reply:
we have an @fluid-internal namespace we use that will not be published

Reply:
We can use '@fluid-internal/*' temporarily until we're ready to publish.  It's the experimental namespace that we specifically want to avoid.

Just FYI - policy-check is our own tool.  We can make it do what we want:

tools\build-tools\src\repoPolicyCheck\handlers\npmPackages.ts

```ts
function packageShouldNotBePrivate(name: string): boolean {
    return (
        name.startsWith("@fluidframework") ||
        name.startsWith("@fluid-example"));
}
```

Reply:
you can also use @fluid-experimental which will be published but we assert has fewer compact guarantees

Reply:
Moved to '@fluid-internal/tree'. Thank you @anthony-murphy and @DLehenbauer. Resolving conversation since the `"private": true` has been preserved.